cmd/lakectl/cmd/fs.go

```diff
@@ -132,7 +132,7 @@ var fsCatCmd = &cobra.Command{
 			DieErr(err)
 		}
 
-		defer body.Close()
+		defer func() { _ = body.Close() }()
```

Comment:
This might actually still hide some (usually minor) errors. I'm not sure that network reads are guaranteed to succeed or fail before completion.

Consider opening a separate issue to report errors here.